src/SourceBuild/content/repo-projects/Directory.Build.props

```diff
@@ -84,6 +84,8 @@
     <!-- Pass locations for assets -->
     <BuildArgs>$(BuildArgs) /p:SourceBuiltAssetsDir=$(ArtifactsAssetsDir)</BuildArgs>
     <BuildArgs>$(BuildArgs) /p:SourceBuiltAssetManifestsDir=$(RepoAssetManifestsDir)</BuildArgs>
+    <!-- TODO: Needed to debug null ref issue. Remove once https://github.com/dotnet/dnceng/issues/3305 is resolved -->
+    <BuildArgs>$(BuildArgs) /p:Features=debug-analyzers</BuildArgs>
```

Comment:
Is this going to flow everywhere are needed or should this be set as an env?

Reply:
This flows all the way into the inner builds.

Reply:
This is going to flow everywhere. I'm not sure of a cleaner way to set this and/or if setting this as an env is feasible.

Reply:
Unfortunately `/p:` in MSBuild is a global final setting. So this would override anyone that is using `<Features>` in their project / props files. That is mostly esoteric copmiler flags so unlikley it's being used actively right now. For the moment think we should go forward with this as is to track down this problem. 

@baronfel, @rainersigwald another case where a version of `-p:` which was _initial value_ not _final value_ would be useful.